concepts/05 Widgets/Map/10 Specify the Provider and Type.md

```diff
@@ -85,6 +139,74 @@ When using maps, you should include an API key that authenticates your applicati
         // ...
     })
 
+##### Vue
+
+    <template>
+        <DxMap
+            :zoom="10"
+            :center="centerCoordinates"
+            provider="bing"
+            :key="authentificationKeys"
+        />
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { DxMap } from 'devextreme-vue/map';
+
+    export default {
+        components: {
+            DxMap
+        },
+        data() {
+            return {
+                centerCoordinates: { lat: 40.749825, lng: -73.987963 },
+                authentificationKeys: {
+                    bing: this.BING_API_KEY,
+                    google: this.GOOGLE_API_KEY,
+                    googleStatic: this.GOOGLE_STATIC_API_KEY
+                }
+            };
+        }
+    }
+    </script>
+
+##### React
+
+    import React from 'react';
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { Map } from 'devextreme-react/map';
+
+    const centerCoordinates = { lat: 40.749825, lng: -73.987963 };
+    const authentificationKeys = {
+        bing: this.BING_API_KEY,
+        google: this.GOOGLE_API_KEY,
+        googleStatic: this.GOOGLE_STATIC_API_KEY
```

Comment:
- These variables have to be declared somewhere.
- What does `this` refer to in this case?

Reply:
- see the thread above
- `this` is excess here. Fixed